### ISSUE TITLE
Does the wizard spell Summon Guns improve rounds?

### ISSUE THREAD

**Issue by Probe1** [d3athrow/vgstation13#14164]
People were talking about how survivors by the by don't try to survive.  They treat it like Highlander and "survive" by killing everyone.

Should we nix summon guns, tweak it, or leave it be?

**Comment by Blithering** [comment]
I say leave it be, It's fun for the people to have a chance to cut loose and does it's job of creating a bit of chaos for the wizard to use as a smokescreen.

**Comment by despotate** [comment]
No one ever gets banned for being too much of an antag as an antag, why wouldn't people use the opportunity to maximum griff? Hell, I doubt anyone would even get banned for suicide bombing themselves as "survivor".

**Comment by Shadowmech88** [comment]
You're not going to be able to make people antags without the possibility they'll murder everyone, so if that's the issue then summon guns should just not create survivors.

That said, I don't think it's a problem in the first place.

**Comment by Probe1** [comment]
You will definitely get antag banned for killing yourself as a survivor @despotate

**Comment by ghost** [comment]
Is this referring to summon guns specifically?  Are you including the survivors from summon magic?  Is this just an issue with survivors from both?  I have no issue with the mechanic, granted, I would never take summon guns when summon magic is available.  Some of my favorite rounds as a wizard has been summoning magic x5 and dead chat/endgame OOC chat always seems as if they really enjoyed their selves and the opportunity.

**Comment by Exxion** [comment]
Summon Guns did nothing wrong

There's no point to giving everyone guns if nobody is allowed to use them. That's why the survivors exist.
And there's nothing wrong with antags deciding to antagonize. If they decide their best bet for survival is to kill everyone else, then whatever. They're wrong, because everyone else also has guns and attempting to murderbone after Summon Guns is a one way ticket to being dead, but there's still nothing wrong with trying.
Honestly, someone attempting to murderbone after Summon Guns is far more interesting than someone murderboning in... basically any other scenario

**Comment by kol1th** [comment]
I don't think that's a problem with survivors or whatever objectives they get, it's more of a problem with "if you're antag you do whatever you want objectives don't matter". 

I'm not saying that's a problem overall, but in this case that would be what you're referring to.

Wizards in general are kind of fucked, as a whole gamemode. The round doesn't end when they die, so people just kill them in five minutes and then it's extended, unless an admin steps in. Even then, an admin could not be online, and then you just have extended where everyone knows nothing will happen for the next however long.  I don't mean to say the round ending like it did before was good, but maybe just doing away with wizard as a whole and replacing it with wizard wars, or having other antags (autotraitor) take over rounds where all of the antags are dead.

Removing, changing, or doing whatever to summon guns wouldn't change anything. The objective means nothing, it's just an antag pass. If they weren't made antagonists, then nobody could use them. I mean, it doesn't even give everyone survivor as is, unless you take it 5 times (best strategy).

Don't forget a wizard puts themselves at a serious risk taking summon guns, since ballistics are so hilariously effective. 

Basically, I don't think changing it is required, or needed, or should be done. Even if it was changed to where they HAD to "survive" or risk being banned, it would only serve to make it HARDER for the wizard who just spent a bunch of spell points on giving everyone guns.

**Comment by TheFriggenSawbones** [comment]
its fun

**Comment by Riekland** [comment]
wait a sec are you trying to erode my second amendment rights again

**Comment by Fruktlimpa** [comment]
@Probe1 How is deliberately ignoring your objective as a survivor any different from deliberately ignoring your objective as a tator?
Having different standards for different antags needs to be clear in that case.

**Comment by SonixApache** [comment]
YES, IT DOES
TURNING HALF THE CREW INTO ANTAGS IS A SURE WAY TO SPICE UP ANY ROUND

**Comment by AI-Is-Rogue** [comment]
Summon magic is superior in my opinion.
So many people slinging spells, you don't know who truly is the wizard anymore and it can cause so much more unique chaos than just shooting people in the halls or blowing up a shard.
